services/net/notification/NotificationService.cs

```diff
@@ -47,7 +49,11 @@ protected override IServiceCollection ConfigureServices(IServiceCollection servi
             .AddChesSingletonService(this.Configuration.GetSection("CHES"))
             .AddSingleton(new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.Email, "") })))
             .AddSingleton<IServiceManager, NotificationManager>()
-            .AddSingleton<ITnoRazorLightEngine, TnoRazorLightEngine>();
+            .AddSingleton<IRazorLightEngine>(
```

Comment:
We don't currently need a wrapper class that only provides the basic initialization code.  We can do that here.